slides/src/content/events.md

```diff
@@ -346,10 +346,10 @@ export class Child {
 
 1. I am the child
 
-2. I am the parent  
```

Comment:
Same comment as earlier - the two end-of-line spaces are to force a line break, so `<br>` here please.